rules/instance-metadata-api-access/rule.metadata.json

```diff
@@ -2,7 +2,8 @@
   "name": "instance-metadata-api-access",
   "attributes": {
     "m$K8sThreatMatrix": "Credential Access::Instance Metadata API",
-    "armoBuiltin": true
+    "armoBuiltin": true,
+    "armoOpa": "true"
```

Comment:
@Daniel-GrunbergerCA 
KS currently filters out controls with "armoOpa": "true".
Testing this control without this configuration makes it work - do you remember why you initially configured it that way? Do you think it is safe to remove the "armoOpa": "true" from this control?